refreshdata.py

```diff
@@ -462,7 +496,9 @@ def main():
 
     # Update data/processed files from latest data/sources files
     # Replace data_qc
```

Comment:
seems that this comment line should be removed too

Reply:
I removed the call by mistake, it's back now.